app/init.php

```diff
@@ -605,6 +605,9 @@ function (mixed $value) {
 $register->set('geodb', function () {
     return new Reader(__DIR__ . '/assets/dbip/dbip-country-lite-2022-06.mmdb');
 });
+$register->set('passwordsDB', function () {
+    return \file_get_contents(__DIR__ . '/assets/security/10k-common-passwords');
```

Comment:
It will be much more efficient to parse the list here as a hashmap with the password used as the key, and then using array_key_exists for very fast validation on this specific data structure.

Reply:
updated